roles/initialize_disks/tasks/main.yml

```diff
@@ -0,0 +1,112 @@
+---
+- name: Retrieving node ID from storpool.conf
+  command: storpool_showconf -n -e SP_OURID
+  register: retrieve_id_result
+  tags:
+    - check_configuration
+
+- name: Checking if host is defined in storpool.conf
```

Comment:
this check is not just for disk roles

Reply:
I moved it back to the stage where requirements are checked